### DISCUSSION TITLE
How to embed content from Storybook

### DISCUSSION THREAD

**Discussion by itsJohnnyGrid** [GitbookIO/community#673] [1. How to]
Hi, GitBook team. 👋

We are developing open-source UI library for Vue.js and decided to use GitBook as a home of our documentation.

I found that GitBook have ability to embed external content by URL (https://docs.gitbook.com/content-editor/blocks/embed-a-url) but when I trying to embed some content from our Storybook it doesn't works and I see block with link only (example of link I tried to use: https://ui.vueless.com/iframe.html?args=&id=1010--default&viewMode=story).
![image](https://github.com/user-attachments/assets/097af980-113d-49d7-bee2-0a5e7a9e35b8)

Same way If I try to add same URL into Medium article, it works well. Example: https://medium.com/@ghengeveld/embedding-storybook-on-medium-ce8a280c03ad

Maybe I need add some config somewhere or it's a GitBook bug / unsupported feature?

Thanks 🙏

**Comment by aikoii** [comment]
👋 @itsJohnnyGrid! I'm not sure if it'll work, but instead of using the iFrame src directly, could you try using its canonical URL?
